net/lnet/ut/lnet_ut.c

```diff
@@ -299,7 +303,7 @@ struct ut_data {
 };
 
 #ifdef __KERNEL__
-M0_BASSERT(UT_PAGE_SHIFT == PAGE_SHIFT);
+//M0_BASSERT(UT_PAGE_SHIFT == PAGE_SHIFT);
```

Comment:
Please add comments to explain this,
or remove it.

Reply:
restored the original statements